test/unit/lib/fullUrlUtils.js

```diff
@@ -0,0 +1,18 @@
+const chai = require('chai');
+const fullUrlUtils = require('../../../app/lib/fullUrlUtils');
+
+const expect = chai.expect;
+
+describe('fullUrlUtils', () => {
+  function host() { return 'beta.nhs.uk'; }
+  const mockReturnUrl = 'https://beta.nhs.uk/find-a-pharmacy/results?location=ls';
+
+  describe('url', () => {
```

Comment:
The test will need refactoring if the suggested changes are implemented.

Reply:
Done